src/mmw/js/src/core/layerPicker.js

```diff
@@ -312,63 +241,11 @@ var LayerPickerView = Marionette.LayoutView.extend({
     },
 
     collectionEvents: {
-        'change': 'render'
+        'change': 'render',
+        'change:layer': 'render'
```

Comment:
I'm still working out my mental model of how these events work in Marionette but: would `change:layer` trigger render 2x, as there's a generic change happening then the specific change event? Does the order here matter?